ros/processor/inventory_events_consumer.py

```diff
@@ -182,6 +182,16 @@ def _extract_spec_file_data(self, tar, file):
         return extracted_file.read().decode('utf-8')
 
     def _calculate_performance_score(self, performance_record):
+        MAX_IOPS_CAPACITY = 0
         memory_score = (float(performance_record['mem.util.used']) / float(performance_record['mem.physmem'])) * 100
-        performance_score = {'memory_score': int(memory_score)}
+        # considering 16000 as the max iops capacity for volumes in AWS/Azure
+        with app.app_context():
+            cloud_query = (db.session.query(System.cloud_provider).filter(System.id == PerformanceProfile.system_id))
+            cloud_provider = cloud_query.first()[0]
+            if cloud_provider == 'aws':
+                MAX_IOPS_CAPACITY = 16000
+            if cloud_provider == 'azure':
+                MAX_IOPS_CAPACITY = 20000
+            io_score = (float(performance_record['disk.all.total']) / float(MAX_IOPS_CAPACITY)) * 100
```

Comment:
MAX_IOPS_CAPACITY is set to default 0, If somehow cloud_provider is null or something else. then `ZeroDivisionError` exception may raise, better to have `else` block to set it to `16000`.

Reply:
Yeah I pushed my non-stashed changes too. Will do. :+1: